### PR TITLE
adds tacolizard takedown but less bad.

### PR DESCRIPTION
**Pull request by ghost**
aight so I liked tacolizard's idea but the stun throw hatchet was an awful awful mechanic that they didn't change

so basically here's what it does now

It's the same as before-Alt click a hatchet to set a warcry, which you scream if you suicide or throw the hatchet at someone.

The hatchet has an embed chance of 50%. I'm concerned that's fairly high, but I can adjust it based on feedback.

Pain multiplier is 3 rather than his chosen 4.

Throw range is six now instead of 5. 

The main mechanic here is that throwing the hatchet on a mob with low health-20 or lower-will stun them. Previously, it was guaranteed regardless of health. Now, it's mostly just a Navajo style takedown on almost dead targets. And before you worry, the hatchet throw did 15 damage before, so if someone had 20 health or lower anyway when you hit them with the hatchet they were basically dead.

Also it no longer gives combat two because what the fuck it's a plastic hatchet

also i am bad at coding this is probably going to fail don't hit me
:cl: Batman
add: The hatchet can now be thrown at weakened, injured targets to deliver a short stun! No, this isn't as bad as it sounds.
add: Alt-click a hatchet to set a war cry.
/:cl:

### CODE REVIEW

**Comment by AdamElTablawy** [comment]
yep called it

**Comment by XDTM** [comment]
Pain multiplier multiplies w_class, so you're adding 3 whole damage when embedding. In my opinion, just raise it a bit, so you deal 20 damage to a guy with 20 health and send him to crit, without needing any special stuns. Even better if you can guarantee embedding on low health.

**Comment by Tacolizard** [comment]
I'm tacolizard and I approve this pr

**Comment by kevinz000** [comment]
good job performing a github takedown on tacolizard

**Comment by AdamElTablawy** [comment]
well, if i'm going to make it a 20 damage weapon outright that's a throwable laser except it embeds and causes bleeding and fits in your pocket

so uh, i'd pass, tbh

unless i increased health to 40 or so, but eh

**Comment by ExcessiveUseOfCobblestone** [comment]
so there were never any sprites for the new hatchet? Just MEMES?

i'm upset.

**Comment by XDTM** [comment]
I mean, lasers have multiple shots with 20 damage each that go further and through windows and can be recharged and can't be thrown back at you. I don't really think they'd be so overpowered.

**Comment by Tacolizard** [comment]
@ExcessiveUseOfCobblestone the hatchet was already in the game

File: code/modules/hydroponics/hydroitemdefines.dm [hunk 1]
```diff
@@ -75,25 +75,65 @@
...
 	playsound(loc, 'sound/weapons/bladeslice.ogg', 50, 1, -1)
 	return (BRUTELOSS)
+	
+/obj/item/weapon/hatchet/AltClick()
+	..()
+	if(ismob(loc))
+		var/mob/M = loc
+		var/input = stripped_input(M,"What do you want your war cry to be? You will shout it when you perform a takedown.", ,"", 50)
+		if(input)
+			src.war_cry = input
```
Comment (optimumtact) [inline code/modules/hydroponics/hydroitemdefines.dm]: you don't need this src at all

File: code/modules/hydroponics/hydroitemdefines.dm [hunk 2]
```diff
@@ -75,25 +75,65 @@
...
+	var/war_cry = "WHOOP WHOOP WHOOP!!"
 
 /obj/item/weapon/hatchet/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is chopping at [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.visible_message("<span class='suicide'>[user] is scalping [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to get closer to their ancestors!</span>")
+	user.say("[war_cry]")
 	playsound(loc, 'sound/weapons/bladeslice.ogg', 50, 1, -1)
 	return (BRUTELOSS)
+	
+/obj/item/weapon/hatchet/AltClick()
```
Comment (optimumtact) [inline code/modules/hydroponics/hydroitemdefines.dm]: is this properly accounting for adjacency and only living mobs handling this item?

i.e can you alt click this as a ghost and at distance

File: code/modules/hydroponics/hydroitemdefines.dm [hunk 3]
```diff
@@ -75,25 +75,65 @@
...
+	..()
+	if(ismob(loc))
```
Comment (optimumtact) [inline code/modules/hydroponics/hydroitemdefines.dm]: why are you limiting it to only when there is a mob?

pass usr into the stripped_input call and you'll always get the person on the other end of the click.

File: code/modules/hydroponics/hydroitemdefines.dm (hunk 1 above)
Comment (Cyberboss) [inline code/modules/hydroponics/hydroitemdefines.dm]: 68747470733a2f2f66696c652e686f7573652f446f54562e706e67

File: code/modules/hydroponics/hydroitemdefines.dm [hunk 4]
```diff
@@ -75,25 +75,65 @@
...
+		var/mob/living/carbon/human/H = L
+		if(H.check_shields(0, "[user]'s [name]", src, MELEE_ATTACK)) //No message; check_shields() handles that
+			playsound(L, 'sound/weapons/Genhit.ogg', 50, 1)
+			return 0
+		else if(L.health < 20) //only takedowns very nearly dead people for balance reasons. 
+			L.Weaken(1)
+			if(user)
+				user.lastattacked = L
+				L.lastattacker = user
+				L.visible_message("<span class='danger'>[user] has done a takedown on [L] with [src]!</span>", \
```
Comment (PKPenguin321) [inline code/modules/hydroponics/hydroitemdefines.dm]: Or "has taken down" depending on what tense you're going for

Comment (Tacolizard) [inline code/modules/hydroponics/hydroitemdefines.dm]: i wrote that. the word takedown is annoyingly past-tense only so i just used dumb grammar.

**Comment by Tacolizard** [comment]
yo buff it back up to force 12. it's a hatchet

**Comment by PKPenguin321** [comment]
yeah regular force doesn't need to be changed

**Comment by Thunder12345** [comment]
Can you make the 20 health a var?

**Pull request closed without merging**